CoreFoundation/Base.subproj/CFUtilities.c

```diff
@@ -764,7 +767,16 @@ void CFLog(CFLogLevel lev, CFStringRef format, ...) {
 #if DEPLOYMENT_RUNTIME_SWIFT
 // Temporary as Swift cannot import varag C functions
 void CFLog1(CFLogLevel lev, CFStringRef message) {
+#ifdef __ANDROID__
+    UInt8 buffer[4096];
+    CFIndex usedBufLen = sizeof buffer-1;
+    CFStringGetBytes(message, CFRangeMake(0, CFStringGetLength(message)),
+		     kCFStringEncodingUTF8, ' ', FALSE, buffer, usedBufLen, &usedBufLen);
+    buffer[usedBufLen] = '\000';
+    __android_log_print(ANDROID_LOG_INFO, "Swift", "%s", buffer);
```

Comment:
Mapped the log levels but I couldn’t find a way to get the app name without reching back to Java through JNI.